index.html

```diff
@@ -0,0 +1,196 @@
+        </ul>
+
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <section class="first-container" aria-label="who I am">
+      <section class="introduction" aria-label="who I am">
+        <h3 class="">"Hello! Sharing world"</h3>
+        <div class="mask">
+          <h1>GRAPHIC DESIGNERS GLOBAL SUMMIT 2023</h1>
+        </div>
+        <p class="short-intro">
+          A joyful celebration believing in the value of openness and sharing,
+          creating a positive with people from over 80 countries is taking place
+          in January, in Nigeria.
+        </p>
+        <p class="time">2023.01.03(TUE) ~ 04(WED)</p>
+        <p class="venue">
+          @ National Theatre and Museum, Art Center Lagos and more
+        </p>
+      </section>
+    </section>
+  </section>
+
+    <!-- Main Program Section -->
+
+    <section class="forum">
+      <h2 class="main-program">Main Program</h2>
+      <span class="line-bar"></span>
+
+      <section class="grid-container" aria-label="summit program">
+        <article class="main-card" aria-label="My Expertise ">
+          <img src="img/lectures1.png" alt="lecture details" />
+          <h3>Lecture</h3>
+          <p>
+            Listen to the speakers from various countries about the message of
+            sharing and opening
+          </p>
+        </article>
+        <article class="main-card" aria-label="summit program ">
+          <img src="img/exhibition1.png" alt="Exhibition details" />
+          <h3>GC Exhibition</h3>
+          <p>
+            Appreciate various creations applying GD license of artist
+            organized, from Art Center Lagos
+          </p>
+        </article>
+        <article class="main-card" aria-label="summit program ">
+          <img src="img/forum1.png" alt="forum details" />
+          <h3>Forum</h3>
+          <p>
+            Have the time to share your thoughts and opinions with experts for
+            each topic
+          </p>
+        </article>
+        <article class="main-card" aria-label="summit program ">
+          <img src="img/workshop1.png" alt="workshop details" />
+          <h3>Workshop</h3>
+          <p>
+            Try creating your own work using open source license rather than
+            just watching at it.
+          </p>
+        </article>
+        <article class="main-card" aria-label="summit program">
+            <img src="img/ignite.png" alt="ignite forum"/>
+          <h3>GC Ignite</h3>
+          <p>
+            Get opportunities to network with GD affiliates around the world,
+            also after the summit
+          </p>
+        </article>
+      </section>
+      <div class="whole-container">
+        <h3 class="whole-program">SEE THE WHOLE PROGRAM</h3>
+        <a href="" class="join-summit"> Join GD Global Summit 2015</a>
+      </div>
+    </section>
```

Comment:
- [ ] [optional] Kindly check the design of your main program, the article's design is breaking after 379px, it would be nice if the design was consistent. 
![image](https://user-images.githubusercontent.com/61408860/210959179-61fb365e-4b11-4fdb-8073-c57d548f2451.png)

Reply:
I have implemented that